agent/command/git.go

```diff
@@ -101,9 +120,9 @@ func (opts cloneOpts) validate() error {
 		catcher.New("missing required location")
 	}
 	if opts.method != "" {
```

Comment:
This looks like it should validate unconditionally (i.e. `opts.method` is required) now that the `switch` on clone method doesn't support the empty string.